mealie/schema/group/group_shopping_list.py

```diff
@@ -177,6 +177,10 @@ class ShoppingListItemsCollectionOut(MealieModel):
     deleted_items: list[ShoppingListItemOut] = []
 
 
+class ShoppingListItemPagination(PaginationBase):
```

Comment:
There really is no need for this, since PaginationBase is generic to begin with and this subclass doesnt add anything in particular. You can just use `PaginationBase[ShoppingListItemOut] ` instead

Reply:
Yeah the only reason I added it is because we have them for all other `get_all` routes, so I wanted to keep it consistent with the rest of the code base